index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Havier medical services survey form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <h1 id="title">HAVIER MEDICAL SERVICES SURVERY FORM</h1>
+        <p id="description">Fill in this survey to help us treat you better.</p>
+
+        <form id="survey-form">
+            <div> 
+               
+                <label id="name-label">Name
+                <input type="text" id="name" placeholder="Enter your name">
+                </label>
+            </div>
+            <div>
+               
+                <label id="email-label">Email
+                <input type="email" id="email" placeholder="Enter your email">
+                </label>
+            </div>
+            <div>
+                <label id="number-label">Age
+                <input type="number" id="number" placeholder="Choose Age" min="18" max="60">
```

Comment:
There was no required keyword at the end of `input tags` fix it.

Reply:
Good work using inline comments to point out exactly where the change should be made.

-  Please always explain why a change is required to the student to give the student some context. 😀 
- You can also let the student know that you are available to offer help and encourage them to reach out in case they do not understand the change request. 👍🏾 
- Whenever needed, please share [links](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation) to help the student better understand why certain features are required 🟢